template.yaml

```diff
@@ -817,9 +875,31 @@ Resources:
         pS3SyncScheduleExpression: "cron(0/30 * ? * * *)"  # default is to sync documents every 30 minutes
         pCustomerManagedEncryptionKeyArn: !GetAtt CustomerManagedEncryptionKey.Arn
         pChunkingStrategy: 'No chunking'
+        pEmbedModel: !Ref KnowledgeBaseEmbeddingModelId
         LogLevel: !Ref LogLevel
         PermissionsBoundaryArn: !Ref PermissionsBoundaryArn
 
+  DOCUMENTS3VECTORSKB:
```

Comment:
Just use the existing bedrockkb template.yaml - make the OSS specific resources conditional, and create new confidional custom resources for the S3Vector store index and KB..   All the data source resources should be the same, right?